frontend/apps/reader/modules/readerview.lua

```diff
@@ -487,6 +493,57 @@ function ReaderView:drawTempHighlight(bb, x, y)
 end
 
 function ReaderView:drawSavedHighlight(bb, x, y)
+    -- set/free note mark sign
+    if self.highlight.note_mark == "sidemark" then
+        if not self.note_mark_sign then
+            self.note_mark_sign = TextWidget:new{
+                text = "\u{F040}", -- pencil
+                face = Font:getFace("smallinfofont", 14),
```

Comment:
Is this fixed font size of 14 fine when the user has a very large or a very small font size ? Or should it be dependant/a factor of the cre font size? But I don't think we have anything equivalent for PDFs ... (or may be only when in reflow mode).

Reply:
OK, it would probably look too big (catching the eyes too much) or too small (we wouldn't notice the pencil) if it was to follow the text size.
So, probably best to keep a fixed size - and 14 if it is fine for you. (Looks like I also use 14 for the pencil in BookMap).